backend/db/migrate/20210507005951_add_columns_to_users.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# In this migration, add columns to Users table
+class AddColumnsToUsers < ActiveRecord::Migration[6.1]
```

Comment:
Wait, didn't you already merge a PR with a migration to add name, phone number, etc in?